content/using_bookwyrm/shelves.md

```diff
@@ -0,0 +1,77 @@
+---
+Title: Shelves & Reading Status
+Date: 2025-06-28
+Order: 2
+---
+
+An integral part of managing your reading is organising your books and their read status.
+On Bookwyrm, you use Shelves for this.
+
+To get to the shelves page, select the 'Your Books' link at the top of the website.
+The tabs are your shelves, as well as an 'All books' view.
+
+## Reading Status Shelves
+
+Bookwyrm comes with four default shelves: To Read, Currently Reading, Read and Stopped Reading.
+These are used to track the reading status of each of your books.
+
+### Updating reading status
+
+When browsing books on Bookwyrm, there is a button below or beside each book, used to update its reading status.
+
+The app expects the flow of `Not shelved` -> `To read` -> `Currently reading` -> `Read`.
+So the default option will always advance the book along this flow.
+Other options are available from the dropdown arrow.
+
+### Only on one status shelf
+
+A book can only be on one of the four reading status shelves at a time.
+When you update your reading progress, the book is moved from one shelf to the next.
+This means that if you're reading a book for a second time, it will be put on Currently Reading and not on Read, even though you had read it before.
+
+### Reading activity dates
+
+When starting and finishing a book, you are prompted to confirm the start and finish dates.
+These can be seen on a books page under the 'Your reading activity' subtitle.
+If you manually add read dates, keep in mind that it won't automatically shelve the book.
+
+### Posting status updates
+
+In the same pop-up screen you confirm reading dates, you have the option to post a reading status update to your feed.
+These will appear in the feeds of people who follow you, and have similar [visibility settings](/privacy-controls.html) as most things in Bookwyrm.
+
+### Finish reading or stop reading?
+
+Use 'Finish reading' when you've finished reading a book.
+
+'Stop reading' is for books you don't want to keep reading, also known as a DNF (did not finish).
+
+## Custom Shelves
+
+Custom shelves are completely separate from the reading status shelves.
+Adding books to them works the same though.
+
+### On multiple custom shelves
+
+As opposed to the built-in reading status shelves, a book can be on multiple custom shelves at once.
```

Comment:
I know it probably seems obvious, but maybe we should spell out that a book can be on a reading status shelf as well as one or more custom shelves simultaneously?